src/MissionManager/MissionController.cc

```diff
@@ -2509,14 +2509,18 @@ void MissionController::_updateTimeout()
                 case MAV_CMD_NAV_WAYPOINT:
                 case MAV_CMD_NAV_LAND:
                     if(pSimpleItem->coordinate().isValid()) {
+                        double alt = 0.0;
```

Comment:
get the current item's altitude, if available. Somehow, `pSimpleItem->coordinate()->altitude()` is incorrect but the altitude in `pSimpleItem->altitude()` is correct.